src/components/Dropdown.js

```diff
@@ -1,47 +1,115 @@
-import React, { useState, useRef } from 'react'
+import React, { useState, useRef, useEffect } from "react";
+import { createUseStyles } from "react-jss";
+import { ContributorThumbnail } from "./ContributorThumbnail";
 
-const dropdown = {
-    margin: '0.4rem 0',
-    boxSizing: 'border-box',
-    width: '100%',
-    display: 'flex',
-    justifyContent: 'space-between',
-    alignItems: 'center',
-    color: '#004364',
-    backgroundColor: 'white',
-    borderRadius: '2px',
-    padding: '0 1rem',
-    boxShadow: '0.1rem 0.1rem 10px #e0dede ',
-};
+const useStyles = createUseStyles({
+  container: {
+    width: "100%",
+  },
+  codeForAll: {
+    "& h5": {
+      color: "#0F1D2F",
+    },
+    margin: "1rem auto",
+    color: "#004364",
+    boxSizing: "border-box",
+    width: "50%",
+    display: "flex",
+    justifyContent: "space-between",
+    alignItems: "center",
+    backgroundColor: "white",
+    borderRadius: "2px",
+    paddingRight: "1rem",
+    "&:nth-of-type(2)": {
+      color: "red",
+    },
+  },
+  dropdown: {
+    "& h5": {
+      color: "#0F1D2F",
+    },
+    maxHeight: '6rem',
+    margin: "0.75rem 0",
+    color: "#004364",
+    boxSizing: "border-box",
+    width: "100%",
+    display: "flex",
+    justifyContent: "space-between",
+    alignItems: "center",
+    backgroundColor: "white",
+    borderRadius: "2px",
+    paddingRight: "1rem",
+    "&:nth-of-type(2)": {
+      color: "red",
+    },
+  },
+  chevron: {
+    marginLeft: "auto",
+    fontSize: "1.3rem",
+    padding: "0.5rem",
+    "&:hover": {
+      cursor: "pointer",
+    },
```

Comment:
You can remove the 'hover' selector from chevron and add the 'cursor: pointer' style along with the other styles; otherwise, for some reason, the cursor never changes to a pointer.
```suggestion
  chevron: {
    cursor: "pointer",
    marginLeft: "auto",
    fontSize: "1.3rem",
    padding: "0.5rem",
```